src/intree.js

```diff
@@ -130,8 +130,12 @@ module.exports.setup = function(cfg) {
 
         let events = task.task.extra.github.events;
         let branches = task.task.extra.github.branches;
+        let branch = payload.details['event.base.repo.branch'];
+        let excludeBranches = task.task.extra.github.excludeBranches;
```

Comment:
Oh, one other thing. If `task.task.extra` doesn't exist this will throw an error!

Reply:
[This](https://github.com/taskcluster/taskcluster-github/blob/master/src/handlers.js#L304) line in src/handlers.js  takes care of it.